layouts/partials/article.html

```diff
@@ -5,7 +5,9 @@
 				</header>
 {{ end }}
 {{ if or (eq $.mode "full") (eq $.mode "body") }}
+				{{ if default (default true (index .Site.Params "toc")) (index .Params "toc") }}
```

Comment:
I think it could be written:
```
{{ if default (default true .Site.Params.toc) .Params.toc }}
```
By the way, you where right. The index operation was not what I thought.